### PR TITLE
milkomeda chain details

### PR DESCRIPTION
**Pull request by bartfish**

### CODE REVIEW

File: _data/chains/eip155-2001.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Milkomeda C1 Mainnet",
+  "chain": "milkAda",
+  "icon": "milkomeda",
+  "rpc": [
+    "https://rpc.c1.milkomeda.com:8545",
```
Comment (nicarq) [inline _data/chains/eip155-2001.json]: @rinor @TheFynx can you confirm

File: _data/chains/eip155-2001.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "wss://rpc.c1.milkomeda.com:8546"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "MilkAda",
```
Comment (nicarq) [inline _data/chains/eip155-2001.json]: can we keep it with a non-capitalized `m` 🙏 `milkADA`

Comment (bartfish) [reply _data/chains/eip155-2001.json]: Of course @nicarq, I'll update it shortly

File: _data/chains/eip155-2001.json (hunk 1 above)
Comment (rinor) [inline _data/chains/eip155-2001.json]:
```suggestion
    "https://rpc-mainnet-cardano-evm.c1.milkomeda.com",
```

File: _data/chains/eip155-2001.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Milkomeda C1 Mainnet",
+  "chain": "milkAda",
+  "icon": "milkomeda",
+  "rpc": [
+    "https://rpc.c1.milkomeda.com:8545",
+    "wss://rpc.c1.milkomeda.com:8546"
```
Comment (rinor) [inline _data/chains/eip155-2001.json]:
```suggestion
    "wss://rpc-mainnet-cardano-evm.c1.milkomeda.com"
```

**Review by rinor — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-2001.json (hunk 2 above)
Comment (NicolasDP) [reply _data/chains/eip155-2001.json]: can you change the `testnet` one too please @bartfish  ?

Comment (bartfish) [reply _data/chains/eip155-2001.json]: ah yes, I'll update this one too

Comment (bartfish) [reply _data/chains/eip155-2001.json]: done

**Review by NicolasDP — APPROVED** [review]
@bartfish can you add the ipfs link. I think we are nearly good to go

**Review by NicolasDP — APPROVED** [review]
my bad. All good

**Review by NicolasDP — CHANGES_REQUESTED** [review]
Wait the icons are supposed to go there: https://github.com/dcSpark/chains/tree/master/_data/icons

**Review by NicolasDP — APPROVED** [review]

**Pull request merged**